src/client/src/routes/StyleguideRoute/mapProps.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copied from @evanrs/map-props
+ * https://github.com/evanrs/map-props
+ */
```

Comment:
I don't think this is appropriate.

Reply:
How so? It would be appropriate without the comment?

Reply:
This specific file only serves `mapMarginPaddingProps` a useful tool for design heavy marketing content — this mimics `styled-system` convention of `px …`, `mx …` for adding white space. Unfortunately Styled System does not support Emotion — otherwise I'd readily advise adopting it here.

And, because the conventions like this rest on a great deal of existing work and experience it will necessarily not be a wholly unique implementation. However, as this code now resides within the code base it poses issue of governance or maintenance that might present a risk to Reactive Trader in the long term.

Reply:
However, `mapMarginPaddingProps` can be rewritten to isolate the implementation there.

Reply:
It is my opinion that we should not be tagging our work on RT. If you're comfortable with leaving it here without your name, I think that's fine. However, if you're not, I would suggest rewriting or finding a replacement. Again, this is my personal opinion and I don't mind if you find someone to overrule this.